src/store/wizardSlice.ts

```diff
@@ -272,6 +276,10 @@ export const selectInputValidation =
     return isValid ? 'success' : 'error';
   };
 
+export const selectFirstBootScript = (state: RootState) => {
+  return state.wizard.firstBoot?.script;
```

Comment:
No need for `?` here I think if we use `''` as the empty/none state.

Reply:
if you edit an old blueprint, `state.wizard.firstBoot` is undefined.